lighthouse-core/audits/uses-rel-preconnect.js

```diff
@@ -90,7 +91,7 @@ class UsesRelPreconnectAudit extends Audit {
           // filter out all resources where timing info was invalid
           !UsesRelPreconnectAudit.hasValidTiming(record) ||
           // filter out all resources that are loaded by the document
-          record.initiatorRequest() === mainResource ||
+          record._initiator.url === mainResource.url ||
```

Comment:
these aren't totally equivalent, fwiw. but the difference may not matter. And I suppose devtools was making this assumption previously. :p

Reply:
yeah I was kinda shocked that the whole initiatorRequest logic is literally just first one with matching URL and manager 😮